website/docs/docs/build/packages.md

```diff
@@ -23,6 +23,12 @@ dbt _packages_ are in fact standalone dbt projects, with models and macros that
 * You can use `ref` in your own models to refer to models from the package.
 * You can use macros in the package in your own project.
 
+:::note Using Python packages
+
+Defining and installing dbt packages is different than defining and installing [Python packages](/docs/build/python-models#using-pypi-packages).
```

Comment:
hyperlinking that action so users are absolutely clear the doc is about defining and installing pypi packages, not just about python.

```suggestion
Defining and installing dbt packages is different from [defining and installing Python packages](/docs/build/python-models#using-pypi-packages).
```